Resources/Prototypes/Datasets/tips.yml

```diff
@@ -123,3 +123,46 @@
   - "Monkeys have a rare chance to be sentient. Ook!"
   - "You can tell if an area with firelocks up is spaced by looking to see if the firelocks have lights besides them."
   - "Instead of picking it up, you can alt-click food to eat it. This also works for mice and other creatures without hands."
+  #------------#
+  #  Frontier  #
+  - "Smaller shuttles offer greater maneuverability, while bigger ones offer a wider and better spectre of services. Choose wisely!"
+  - "If you're low on funds, try wandering into nearby ships and ask the local captains if they need a hand. Perhaps you can pick up a high-paying job!"
+  - "It's often better to join an existing crew than to become a solo captain. It can teach you new things and give you a good sum of money."
+  - "It's always better to have a crew than to stay alone in deep space. A friend can save your life if you make a mistake, or save you from trouble!"
+  - "Scientists often sell interesting technologies that can assist you in your job. Perhaps you would want a bottomless bag that can fit an entire station?"
+  - "As a scientist, you can sell your technologies to others. Many would want a bluespace bag, or a chemical dispenser."
+  - "Felinids and other small people can sit in bags! Ask around - maybe there are feline friends seeking for a new home?"
+  - "While holding a bag with 120 free space, you can force a small person inside. Be careful, they may get mad at you!"
+  - "Staying in deep space induces severe stress and often leads to a sudden sleep disorder. You can hire an emotional support crewmember to help you overcome it!"
+  - "You can use your hands to carry people! Alt-click a person while having two free hands, or use the context menu to do so."
+  - "The Vulpkanin will love you if you keep petting them."
+  - "When you enter cryosleep, your body remains stored for up to 60 minutes! As a ghost, you can use the 'un-cryo' button to wake up and return to it!"
+  - "If you accidentally left the game for a long time, you can still check on your character. Perhaps someone shoved them into a cryopod and you can use the 'un-cryo' button?"
+  - "If you see a person who's fallen SSD, make sure to drag them to cryo and shove them into a cryopod. It will prevent them from dying, and will allow the player to return to the body later!"
+  - "Vending machine food is bad for your health! Watch out for food trucks such as the Skipper or the McCargo - they can offer delicious food for a reasonable price!"
```

Comment:
maybe include another note about how Chef cooked food has more flavor, and will give you a speed buff

Reply:
I dunno, can you give me an example?

Reply:
"Meals cooked by chefs are more filling, more flavorful, and can even boost your movement speed for a short time"

Reply:
Yea, added it